src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  font-style: normal;
+  text-transform: uppercase;
```

Comment:
it's not a good practice to set text-transform: uppercase; for the whole document